x-pack/plugins/maps/public/classes/layers/layer.tsx

```diff
@@ -49,8 +48,6 @@ export interface ILayer {
   supportsFitToBounds(): Promise<boolean>;
   getAttributions(): Promise<Attribution[]>;
   getLabel(): string;
-  getCustomIconAndTooltipContent(): CustomIconAndTooltipContent;
-  getIconAndTooltipContent(zoomLevel: number, isUsingSearch: boolean): IconAndTooltipContent;
```

Comment:
+1 always nice to see an interface slim down. it will help maintenance down the line.